devops/azure-pipelines/gradle-build-test-stage-template.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  - name: 'jobName'
+    type: string
+  - name: 'testPath'
+    type: string
+  - name: 'ccPath'
```

Comment:
the testPath and ccPath parameters should have a comment to document them, so it's clear to the user what they should be without having to look at the step definition below.
same applies to the other template file. for the var group in particular, document which variables are expected to be in the var group. or, arguably better, let the pipeline manage the var group, and just pass references to the two variables taken from the group as parameters instead of the group name.